lib/cisco_node_utils/vlan.rb

```diff
@@ -206,5 +224,86 @@ def mapped_vni=(vni)
     def default_mapped_vni
       config_get_default('vlan', 'mapped_vni')
     end
+
+    # Helper method to delete @set_args hash keys
+    def set_args_keys_default
+      @set_args = {}
+    end
+
+    def private_vlan_type
+      result = config_get('vlan', 'private_vlan_type_all')
```

Comment:
Not sure I got your comment. When we do show run private-vlan type, we are not under any vlan context ( like in config time) so how do i get the vlan id @vlan_id?